resources/definitions/eazao_zero.def.json

```
@@ -0,0 +1,127 @@
+{
+    "version": 2,
+    "name": "Eazao Zero",
+    "inherits": "fdmprinter",
+    "metadata": 
+  {
+        "visible": true,
+        "author": "Eazao",
+        "manufacturer": "Eazao",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_quality": true,
```

```suggestion
        "has_machine_quality": true,
```
This PR does not provide specialised qualities for this machine, so it should be false.